src/ui/components/Tag/Tag.tsx

```diff
@@ -16,28 +20,45 @@ const LINE_HEIGHT = getSpacing(NUMBER_OF_SPACES_LINE_HEIGHT)
 
 export const TAG_HEIGHT = PADDING_VERTICAL + LINE_HEIGHT + PADDING_VERTICAL
 
-export const Tag: FunctionComponent<TagProps> = ({ label, Icon, ...props }) => {
+export const Tag: FunctionComponent<TagProps> = ({
+  label,
+  Icon,
+  backgroundColor,
+  paddingHorizontal,
+  ...props
+}) => {
+  const renderIcon = () => {
+    if (!Icon) return null
+
+    if (isReactElement(Icon)) {
+      return cloneElement(Icon, { testID: 'tagIcon' })
+    }
+    if (isStyledComponent(Icon)) {
+      return <Icon testID="tagIcon" />
+    }
+
+    return null
+  }
+
   return (
-    <Wrapper {...props}>
-      {Icon ? (
-        <IconContainer>
-          <Icon testID="tagIcon" />
-        </IconContainer>
-      ) : null}
+    <Wrapper backgroundColor={backgroundColor} paddingHorizontal={paddingHorizontal} {...props}>
```

Comment:
```suggestion
    <Wrapper {...props}>
```
On veut pas enlever ces paramètres ici vu qu'ils sont définis plus bas ?